src/main/java/uk/gov/ons/census/casesvc/model/dto/Address.java

```diff
@@ -19,4 +19,5 @@
   private String addressLevel;
   private String estabType;
   private String organisationName;
+  private boolean secureType;
```

Comment:
I've been testing... it seems like this is fatal if there's no `secureType` supplied in the JSON, or a null is supplied.

In other words, what value is it supposed to use if it's not `true` or `false`? Like `null` or no value present at all? What then? So far as I can see... it blows up pretty badly.

Reply:
actually it might be ok...

Reply:
okay, so the primitive thing does default to `false` but in that sense, we have no idea whether a value was supplied or not. This doesn't meet the requirements.